packages/app-mobile/ios/PrivacyInfo.xcprivacy

```diff
@@ -35,20 +35,12 @@
 			See https://github.com/itinance/react-native-fs/issues/1232 for details. 
 		-->
 		<dict>
-			<key>NSPrivacyAccessedAPITypeReasons</key>
-			<array>
-				<string>C617.1</string>
-			</array>
 			<key>NSPrivacyAccessedAPIType</key>
-			<string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
```

Comment:
The `NSPrivacyAccessedAPICategoryFileTimestamp` API was present twice in this file. XCode automatically removed the second occurrence.